log4j2/src/main/java/com/google/common/flogger/backend/log4j2/Log4j2SimpleLogEvent.java

```diff
@@ -80,21 +85,32 @@ Level getLevel() {
   }
 
   LogEvent asLoggingEvent() {
-    // The Mapped Diagnostic Context (MDC) allows to include additional metadata into logs which
-    // are written from the current thread.
+    // We do not support 'MDC.getContext()' and 'NDC.getStack()' and we do not make any attempt 
+    // to merge Log4j2 context data with Flogger's context data. Instead, users should use the 
+    // ScopedLoggingContext (Grpc).
+    //
+    // Flogger's ScopedLoggingContext allows to include additional metadata and tags into logs 
+    // which are written from current thread.
     //
     // Example:
-    //  MDC.put("user.id", userId);
-    //  // do business logic that triggers logs
-    //  MDC.clear();
+    // try (ScopedLoggingContext.LoggingContextCloseable ctx = GrpcContextDataProvider
+    //          .getInstance()
+    //          .getContextApiSingleton()
+    //          .newContext()
+    //          .withMetadata(COUNT_KEY, 23)
+    //          .withTags(Tags.builder()
+    //              .addTag("foo")
+    //              .addTag("baz", "bar")
+    //              .addTag("baz", "bar2")
+    //              .build())
+    //          .install()) {
     //
-    // By using '%X{key}' in the ConversionPattern of an appender this data can be included in the
-    // logs.
+    //      // do business logic that triggers logs
+    // }
     //
-    // We could include this data here by doing 'MDC.getContext()', but we don't want to encourage
-    // people using the log4j specific MDC. Instead this should be supported by a LoggingContext and
-    // usage of Flogger tags.
-    Map<String, String> mdcProperties = Collections.emptyMap();
+    // By using '%X{key}' in the ConversionPattern of an appender the metadata can be included in the
+    // logs. By using '%x' in the ConversionPattern of an appender the tags can be included in the logs.
+    ContextDataProvider contextDataProvider = Log4j2ContextDataProvider.getInstance();
```

Comment:
This should just be "ContextDataProvider.getInstance();"

Reply:
done